app/assets/stylesheets/components/_autocomplete.scss

```diff
@@ -1,4 +1,4 @@
-@import url(asset-path("accessible-autocomplete/dist/accessible-autocomplete.min.css"));
```

Comment:
The old sass compiler interpreted these as CSS `@import` statements, which is why we had to use this twisted approach. The new one seems to interpret the import in an expected way. Great to see this hack removed.